src/contract-abi.ts

```diff
@@ -0,0 +1,336 @@
+import * as _ from 'lodash';
+
+import {
+  ClarityValue,
+  uintCV,
+  intCV,
+  contractPrincipalCV,
+  standardPrincipalCV,
+  noneCV,
+  bufferCV,
+  falseCV,
+  trueCV,
+  ClarityType,
+  getCVTypeString,
+} from './clarity';
+import { ContractCallPayload, createContractCallPayload } from './payload';
+
+class NotImplementedError extends Error {
+  constructor(message: string) {
+    super(message);
+    this.message = message;
+    this.name = this.constructor.name;
+    Error.captureStackTrace(this, this.constructor);
```

Comment:
I don't think this function is generally available outside of node.js runtime. For libs designed to run in the browser, I usually see this pattern
```ts
if (Error.captureStackTrace) {
  Error.captureStackTrace(this, this.constructor);
}
```

Reply:
Would be cool if there's a CI / lint tool that can error on browser/node specific globals/fns being used. Had a look for isomorphic linters but didn't find much.

Reply:
This should be something flagged if we have the right TSConfig, right?

Reply:
I think once the @node/types package gets added, not much can be done by other tooling to flag things like this. Perhaps the lib should remove the @node/types package, and include any node.js libs manually (e.g. Buffer)